services/api/mw2pdf/utils/mediaWiki.js

```diff
@@ -0,0 +1,61 @@
+import fetch from 'node-fetch'
```

Comment:
Would be nice to write tests for all these utility methods, but we don't have tests set up yet.  Once #20 is done lets come back and fix that deficit.